internal/oci/runtime_vm.go

```diff
@@ -177,7 +177,9 @@ func (r *runtimeVM) startRuntimeDaemon(c *Container) error {
 		r.ctx,
 		newRuntimePath,
 		"",
+		"",
 		c.BundlePath(),
+		nil,
```

Comment:
/cc @sboeuf

Reply:
I"m a bit confused by this change and your comment (or was it the bot's comment?).

Reply:
Hm no, I was just wondering if the containerd API changes somehow have any (maybe bad) influence on the runtime_vm implementation. I would not suggest it, but added a pointer to it :)